llvm/test/CodeGen/AArch64/aarch64-neon-faminmax.ll

```diff
@@ -0,0 +1,207 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc -mtriple=aarch64 -mattr=+faminmax -verify-machineinstrs %s -o - | FileCheck %s
+; RUN: llc -mtriple=aarch64 -verify-machineinstrs %s -o - | FileCheck %s --check-prefix=CHECK-NO-FAMINMAX
+
```

Comment:
There's no great consistency here but do you mind adding `target triple = "aarch64-unknown-linux-gnu"` here and then you can remove the `-mtriple` from the RUN lines.

Reply:
I have and always will disagree with this.  I think RUN lines should be minimised to show the relevant part of the testing.  If a test depends on multiple triples then sure add them to the RUN line.  If, like it is for the majority of cases, the triple is not relevant then it doesn't belong on the RUN line.

Reply:
Sigh, I shouldn't have said anything. The more important part was about adding fminnum support. I don't believe this makes things simpler though, and we have several thousand tests that rightly put the triple in the run line. At the very least it keeps everything close together (triple and attibutes) which makes then easier to read, and in the end of the day being consistent is more important. We will always need triples in some run lines, so it's best to keep them in all. Thanks.